cmd/config.go

```diff
@@ -47,13 +56,21 @@ func newConfigCommand() *cobra.Command {
 	return configCmd
 }
 
+// The PATH arg is actually optional, in order to support compatibility with the --path flag.
 func validateConfig(cmd *cobra.Command, args []string) error {
-	configPath := defaultConfigPath
+	path := defaultConfigPath
```

Comment:
Could potentially extract the code for resolving against the default path into it's own function?

Reply:
Only validate allows a default, for compatibility reasons, so there is no point in refactoring it out.